components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/virtualized-visible-assets-list.tsx

```diff
@@ -101,27 +154,43 @@ export const VirtualizedVisibleAssetsList = (props: VirtualizedTokensListProps)
   }, [])
 
   return (
-    <List
-      ref={listRef}
-      width={'100%'}
-      height={tokenListHeight}
-      itemData={tokenList}
-      itemCount={tokenList.length}
-      itemSize={getSize}
-      overscanCount={10}
-      itemKey={getListItemKey}
-      children={({ data, index, style }) => (
-        <ListItem
-          data={data[index]}
-          isRemovable={isRemovable}
-          onRemoveAsset={onRemoveAsset}
-          isAssetSelected={isAssetSelected}
-          onCheckWatchlistItem={onCheckWatchlistItem}
-          setSize={setSize}
-          index={index}
-          style={style}
-        />
-      )}
-    />
+    <AutoSizer
+      style={{
+        flex: 1
+      }}
```

Comment:
Either wrap with a style component, or declare this style object at module scope (outside of the component) to prevent rerenders

Reply:
Moved!